### ISSUE TITLE
Internal error command

### ISSUE THREAD

**Issue by ghost** [jrbudda/Sentry#244]
Everyone time I try to configure the Sentry I get:http://gyazo.com/40cd099a1f696d98a00fca80b005e1ef
I've looked into the console and this is what I get. Thanks in advance :+1: http://pastebin.com/qJf6cMdX

**Comment by mcmonkey4eva** [comment]
All I see is errors caused by `FairyTailCraft`

**Comment by ghost** [comment]
Well, any ideas on How I can fix it?

**Comment by mcmonkey4eva** [comment]
... ask FairyTailCraft, not us?

**Comment by ghost** [comment]
No, whenever i type in /sentry health 100, it says it Has set the health But it hasn't.

**Comment by mcmonkey4eva** [comment]
Well the errors you see are from FairyTailCraft, not us.

**Comment by ghost** [comment]
So should I remove it?

**Comment by mcmonkey4eva** [comment]
I have no idea what you should do with FairyTailCraft - I have no knowledge of it.
